gcc/rust/hir/rust-ast-lower-expr.cc

```diff
@@ -837,10 +923,22 @@ ASTLoweringExpr::visit (AST::InlineAsm &expr)
 				 mappings.get_next_hir_id (crate_num),
 				 mappings.get_next_localdef_id (crate_num));
 
+  std::vector<HIR::InlineAsmOperand> hir_operands;
+  std::vector<AST::InlineAsmOperand> ast_operands = expr.get_operands ();
```

Comment:
```suggestion
  const std::vector<AST::InlineAsmOperand> &ast_operands = expr.get_operands ();
```

we should be able to just iterate on references

Reply:
resolve :)